yarn.lock

```diff
@@ -2,26 +2,24 @@
 # yarn lockfile v1
 
 
+"@aashutoshrathi/word-wrap@^1.2.3":
```

Comment:
It seems that somehow the yarn.lock got messed, could you try to fix it? One way to solve it can be copying the yarn.lock that is already in `develop`, and run `yarn install` again, make sure that the diffs are only the ones corresponding to the `userEvent` library. After doing this. please feel free of making a git squash of the last 3 commits so that the blame does not point to you.

Reply:
Got it. I have resolved the issue with the `yarn.lock`. I tried squashing the last few commits with rebase, but I ran into a couple of merge conflicts, so I bailed on it. Do we still need the squashing given now the diff in `yarn.lock` is only limited to `userEvent`?

Reply:
Yes, although the PR shows the diff only of the user event change, the blame of almost the whole yarn.lock is still on you because the change was made in a previous commit. But no problem, I can squash it in the PR merge.

I'll give it one last check, and it'll be ready to merge. Thank you! :open_hands:

Reply:
Ahh got it. I'll try to be more careful the next time! Thanks a lot for the guidance and help!